src/playbooks/_tuning/metadata.obsah.yaml

```diff
@@ -0,0 +1,11 @@
+---
+variables:
+  tuning:
+    help: Tuning profile to apply. Defaults to development. Set to a different value according to the available hardware.
```

Comment:
Shouldn't this default to...default?

Reply:
At this point I didn't want to change the current default and impact developers or CI .

Reply:
Can we get this in and I'll change the default to `default` in a subsequent PR and `--tuning development` in the tests to not break them?